BuildPropertiesPlugin/plugin/src/main/groovy/com/novoda/buildproperties/BuildProperties.groovy

```diff
@@ -0,0 +1,41 @@
+package com.novoda.buildproperties
+
+class BuildProperties {
+
+    final String name
+    File file
+    Properties entries
+
+    BuildProperties(String name) {
+        this.name = name
+    }
+
+    String getName() {
```

Comment:
I want to enforce only getters. `file()` as method is needed to achieve fluent API in the `buildProperties` configuration.